### DISCUSSION TITLE
Bootstrap5 Accordion not expanding/collapsing

### DISCUSSION THREAD

**Discussion by RealityMasque** [Megabit/Blazorise#5011] [General]
I found the issues & discussions about needing to use Blazorise.Bootstrap5 in order for the new AccordionToggle to replace using a button.  I copied & pasted from the Blazorise site's documentation for the accordion (https://blazorise.com/docs/components/accordion), but the header, while styled correctly, doesn't expand/collapse.

**Comment by David-Moreira** [comment]
If something is not working as expected. Might be better to open an issue.
If you followed the instructions, well need a reproducable to figure out why it is not working.

**Reply by David-Moreira** [reply]
I will be closing this discussion as no further feedback was provided. Please feel free to reopen or open an issue as suggested.
